plugins/aws-appmesh/plugin/commands.go

```diff
@@ -316,3 +323,27 @@ func (plugin *Plugin) deleteEgressRules(iptable *iptables.IPTables) error {
 
 	return nil
 }
+
+func eachNSlice(ss []string, n int, f func([]string) error) error {
```

Comment:
Nice! Would the code read better if we called this function "forEachNSlice" or "forEachSlice"? We could also rename "n" to "size" or "sliceSize" to improve readability.